src/modules/Generator.jl

```diff
@@ -0,0 +1,86 @@
+"""
+Provides the functions related to generating documentation stubs.
+"""
+module Generator
+
+"""
+    savefile(f, root, filename)
+
+Attempts to save a file at `\$(root)/\$(filename)`. `f` will be called with file
+stream (see [`open`](http://docs.julialang.org/en/latest/stdlib/io-network/#Base.open)).
```

Comment:
Not totally relevant here at the moment of course, but would be cool to be able to use `@ref` across different packages, not sure the nicest way to do that though.